service/controller/controllercontext/status_types.go

```diff
@@ -121,10 +116,9 @@ type ContextStatusTenantClusterTCCPVPC struct {
 }
 
 type ContextStatusTenantClusterTCCPN struct {
-	EtcdVolumeSnapshotID string
```

Comment:
why is etcdSnapshotID  removed in  encryption key PR?

Reply:
There was apparently a duplicate which was not used anywhere. There is still the snapshot ID carried, which is in fact being used.